src/acts.js

```diff
@@ -387,6 +387,62 @@ export default {
             .then(parseDateFields);
     },
 
+    /**
+     * Runs the latest build of given act.
+     *
+     * @memberof ApifyClient.acts
+     * @instance
+     * @param {Object} options
+     * @param {String} options.actId - Unique act ID
+     * @param options.token
+     * @param {String} options.runId - ID a an actor run to metamorph.
+     * @param {String} options.targetActorId - ID of an actor to which run should metamorph.
+     * @param {String|Buffer} [options.body] - Act input, passed as HTTP POST payload
+     * @param {String} [options.contentType] - Content type of act input e.g 'application/json'
+     * @param {String} [options.build] - Tag or number of the build to run (e.g. <code>latest</code> or <code>1.2.34</code>).
+     * @param callback
+     * @returns {ActRun}
+     */
+    metamorphRun: (requestPromise, options) => {
+        const { baseUrl, token, actId, runId, targetActorId, contentType, body, build } = options;
+
+        checkParamOrThrow(baseUrl, 'baseUrl', 'String');
+        checkParamOrThrow(token, 'token', 'String');
+        checkParamOrThrow(actId, 'actId', 'String');
+        checkParamOrThrow(runId, 'runId', 'String');
+        checkParamOrThrow(targetActorId, 'targetActorId', 'String');
+        checkParamOrThrow(contentType, 'contentType', 'Maybe String');
+        checkParamOrThrow(build, 'build', 'Maybe String');
+
+        const safeActId = replaceSlashWithTilde(actId);
```

Comment:
Perhaps use `actorId` and `safeActorId`.

We can do:

```js 
const { /* args */ actId: actorId /* args */ } = options;
```

And only replace this single assignment when we change the naming.